aos/closed_loop/ClosedLoopSimulation.ipynb

```diff
@@ -0,0 +1,448 @@
+{
```

Comment:
<p>Would be nice to put in this place information about when it was run and using what software version, and machine size (cores, memory). Eg.</p><p>*Stack release: <strong>2021_13</strong>&nbsp;Last verified to run: <strong>2021-04-01</strong></p><p>*ts_phosim tag v1.4.0</p><p>...</p><p>etc.</p><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-ts/ts_analysis_notebooks/pull/14/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
<p>Solved</p>